static/sass/_pattern_cube-progression.scss

```
@@ -0,0 +1,201 @@
+@mixin ubuntu-p-cube-progression {
+  $diamondSize: (
+    "width": 168,
+    "height": 99,
+  );
+  $rhombusSize: (
+    "width": 85,
+    "height": 145,
+  );
+
+  .p-cube-progression {
+    height: 382px;
+    position: relative;
+
+    %diamond-badge {
+      background-size: map-get($diamondSize, "width") * 1px
+        map-get($diamondSize, "height") * 1px;
+      height: map-get($diamondSize, "height") * 1px;
+      position: absolute;
+      width: map-get($diamondSize, "width") * 1px;
+    }
+
+    %rhombus-badge {
+      background-size: map-get($rhombusSize, "width") * 1px
+        map-get($rhombusSize, "height") * 1px;
+      height: map-get($rhombusSize, "height") * 1px;
+      position: absolute;
+      width: map-get($rhombusSize, "width") * 1px;
+    }
+
+    &.is-architecture {
+      @extend %diamond-badge;
+
+      background: url(https://assets.ubuntu.com/v1/8b175afd-Architecture.svg);
```

```suggestion
      background-image: url(https://assets.ubuntu.com/v1/8b175afd-Architecture.svg);
```

If you're only specifying the background image, you should be explicit otherwise it's possible to override rules further up the cascade using the shorthand.

---

(The applies throughout this file)